php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/CheckContext.java

```diff
@@ -47,7 +48,12 @@
    * @param message message of the issue
    * @return instance of Issue
    */
-  Issue newIssue(PHPCheck check, String message);
+  @Deprecated
+  PHPIssue newIssue(PHPCheck check, String message);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)